exonum-java-binding-fakes/src/test/java/com/exonum/binding/fakes/mocks/ThrowingTransactionsTest.java

```diff
@@ -60,5 +69,27 @@ public void createThrowingFailsIfUninstantiableThrowable() {
     ThrowingTransactions.createThrowing(UninstantiableException.class);
   }
 
+  @Test
+  @Ignore // Unfortunately, we do not perform such checks at the moment: ECR-1988
```

Comment:
should `TODO` be added before tiket number?

Reply:
Not necessarily :-)